c7n/resources/s3.py

```diff
@@ -1310,6 +1310,68 @@ def process_bucket(self, bucket):
         return {'Name': bucket['Name'], 'State': 'PolicyRemoved', 'Statements': found}
 
 
+@actions.register('set-bucket-replication')
```

Comment:
was just thinking the `bucket` in here is actually redundant

Reply:
Ah yes I guess it redundant because bucket is already implied. Will update in the next commit.

Reply:
Updated in 68908eca5f0db5563bd16c2711546f2f77a756c9